stablehlo.patch

```diff
@@ -0,0 +1,145 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
```

Comment:
this is needed to be able to `bazel test @stablehlo//...` from `mlir-tcp` (can be removed once https://github.com/openxla/stablehlo/pull/1810 lands)